src/app/clusters/level-control/level-control.cpp

```diff
@@ -545,7 +545,7 @@ static void writeRemainingTime(EndpointId endpoint, uint16_t remainingTimeMs)
             markDirty = MarkAttributeDirty::kYes;
         }
 
-        Attributes::RemainingTime::Set(endpoint, state->quietRemainingTime.value().ValueOr(0), markDirty);
+        Attributes::RemainingTime::Set(endpoint, state->quietRemainingTime.value().Value(), markDirty);
```

Comment:
Is this guaranteed to be set now? Before this would default to 0 is has_value is false, but now if you don't have a value here, you'll crash.

Reply:
Yes, It is set above line 543. 
Because `quietRemainingTime.mValue` is init as null, I had initially used ValueOr(0). But it was pointed out in an after-merge review that this will never be null because we just set it to `remainingTimeDs`